### PR TITLE
Create order

### PR DESCRIPTION
**Pull request by mdAlvarenga**

### CODE REVIEW

File: src/main/java/com/edu/unq/tip/enlatada/order/BeerCansCalculator.java [hunk 1]
```diff
@@ -3,10 +3,10 @@
 import org.springframework.stereotype.Component;
 
 /**
- * Calculates beer can using a, at the moment, a constant for represent the beer can capacity
+ * Calculates beer cans using a, at the moment, a constant for represent the beer can capacity
  */
 @Component
-public class BeerCanCalculator {
+public class BeerCansCalculator {
```
Comment (ivigbe) [inline src/main/java/com/edu/unq/tip/enlatada/order/BeerCansCalculator.java]: Can we think a better name? We know that the cans contains beer 😆

Comment (mdAlvarenga) [reply src/main/java/com/edu/unq/tip/enlatada/order/BeerCansCalculator.java]: :es: :speech_balloon: :joy:

Comment (mdAlvarenga) [reply src/main/java/com/edu/unq/tip/enlatada/order/BeerCansCalculator.java]: No se me ocurre otro :(

Comment (ivigbe) [reply src/main/java/com/edu/unq/tip/enlatada/order/BeerCansCalculator.java]: Bueno, dejemoslo asi entonces. Aprobado ;)

**Pull request merged**